docs/source/serialization.mdx

```diff
@@ -149,6 +145,8 @@ DistilBERT we have:
 ["last_hidden_state"]
 ```
 
+The approach is similar for TensorFlow models.
```

Comment:
For this particular example of 

> Hub model checkpoint -> ONNX -> ONNX Runtime

the code is identical for PyTorch and TensorFlow because `transformers.onnx` automatically detects which framework the model weights are loaded in. And once you have the ONNX model, you're free to choose which accelerator to run it on.

What we could do is explicitly load and save a TensorFlow model with e.g. 

```python
from transformers import AutoTokenizer, TFAutoModelForSequenceClassification

# Load tokenizer and TensorFlow model form the Hub
tokenizer = tokenizer = AutoTokenizer.from_pretrained("distilbert-base-uncased")
tf_model = TFAutoModelForSequenceClassification.from_pretrained("distilbert-base-uncased")
# Save to disk
tokenizer.save_pretrained("local-tf-checkpoint")
tf_model.save_pretrained("local-tf-checkpoint")
# Export TensorFlow checkpoint to ONNX
python -m transformers.onnx --model=local-tf-checkpoint onnx/
```

This has the nice feature of showing what's needed to export a local model.

@Albertobegue I realise this has been a long PR, so I'm happy to take care of this detail in a separate PR if you want :)

Reply:
Good idea! Thank you :)